### DISCUSSION TITLE
Pi-Hole Web Interface Won't Load

### DISCUSSION THREAD

**Discussion by MilitaryNerd** [unifi-utilities/unifi-common#368] [Q&A] [answered]
Hi, I was wondering what steps I can take for troubleshooting the web interface? I managed to get it IP'd after an issue; apparently you can't use a locally-administered MAC address (which I had specifically configured because that's what they're for). After I got it to pick up its IP, it will work for resolution but I can't manage it since the web page won't load. Are there any logs I can check for possible errors? Thanks!

**Comment by gatesry** [comment, marked as answer]
You can view podman logs through this command `podman logs -f $CONTAINER_NAME`
